demisto_sdk/yaml_tools/content_creator.py

```diff
@@ -57,6 +60,12 @@ def __init__(self, artifacts_path: str, content_bundle_path='',
         self.file_name_max_size = 85
         self.long_file_names = []  # type:List
 
+    def run(self):
```

Comment:
should have indication of failures (return)

Reply:
i think that it will just fail here